lib/node/runner/api/api.go

```diff
@@ -1,7 +1,7 @@
 package api
 
 import (
-	"boscoin.io/sebak/lib/common/observer"
+	o "boscoin.io/sebak/lib/common/observer"
```

Comment:
Personally I prefer the package name used as is, but I can understand if you don't.

However, single letters identifiers are easy to miss, and very hard to look for in the code, and usually used for variables. Can we get something a tiny bit longer, e.g. `obs` ?